### ISSUE TITLE
Dungeon finder lock

### ISSUE THREAD

**Issue by w5860363** [TrinityCore/TrinityCore#14705]
https://github.com/TrinityCore/TrinityCore/issues/14152
To confirm this problem
test from https://github.com/TrinityCore/TrinityCore/commit/5bd7c295c0b0f2cb78345ed02dd37531a67348f5
DB:TDB58
UPdate:2015_05_12_03_world.sql
No custom

Test when all copies lock will not be able to join the queue which I think is a BUG

Please excuse my English.

http://www.wowwiki.com/Dungeon_Finder

**Comment by Nyeriah** [comment]
In other words: when you're saved for every instance you can't queue to RHC anymore currently. That's indeed a bug.

**Comment by ariel-** [comment]
I remember reading that Dungeon Finder doesn't save the player to the instance in retail, is this true?

**Comment by Nyeriah** [comment]
Yeah it is. But it has worked differently at WotLK (I don't remember exactly how now)

**Comment by Aokromes** [comment]
On 3.3.5a you still was locked into dungeon, and only when you had lock to all instances you was allowed to "repeat"
on 4.3.4 a bug was present giving you lock on well of eternity.

**Comment by mayaren888** [comment]
confirm

**Comment by Aokromes** [comment]
still bug.

**Comment by Rushor** [comment]
suggestion: clear all locks to all lfg dunegons if all lfg dungeons have got an id?

**Comment by w5860363** [comment]
Now it's like
When you lock one, it is excluded from the list and cannot be repeated for that instance. When all are locked, you cannot join the queue.
